src/apps/sc/scATTRACT.cpp

```diff
@@ -20,6 +21,12 @@ class scATTRACT<CPUEnergyService6D<float>>;
 template
 class scATTRACT<CPUEnergyService6D<double>>;
```

Comment:
There is no CUDA awareness in this file anymore in the current version of the master. Conflicts!!